source/deployment/sso-gitlab.rst

```diff
@@ -30,7 +30,7 @@ Keep the GitLab window open because you need the *Application Id* and *Secret* w
 Step 2: Configure Mattermost for GitLab SSO
 -------------------------------------------
 
-1. On your Mattermost server, add the *Application Id* and the *Secret* to the *GitLab* settings section in the ``config.json`` file:
+1. From **System Console > Authentication > OAuth 2.0** select ``Gitlab`` in the *Select OAuth 2.0 service provider* dropdown and add other required fields.  Alternativley, on your Mattermost server, add the *Application Id* and the *Secret* to the *GitLab* settings section in the ``config.json`` file:
```

Comment:
In non-licensed servers, GitLab is listed as a separate section in **System Console > Authentication > GitLab**

Reply:
Thanks!